sound/soc/sof/ipc4-topology.c

```diff
@@ -2808,6 +2873,16 @@ static int sof_ipc4_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget
 		msg = &src->msg;
 		break;
 	}
+	case snd_soc_dapm_asrc:
+	{
+		struct sof_ipc4_asrc *src = swidget->private;
```

Comment:
-> `struct sof_ipc4_asrc *asrc`

Reply:
Yep, I'll do the rename for all "src" with asrc.